managers/opportunity/index.js

```diff
@@ -41,6 +41,146 @@ class opportunityManager {
 
   async getOpportunities(queryObject) {
     try {
+      // Filter based on opportunityType
+      if (queryObject.type) {
+        queryObject['opportunityType'] = queryObject.type;
+        delete queryObject.type;
+      }
+      if (queryObject.female) {
+        queryObject['onlyForFemale'] = queryObject.female === 'true';
```

Comment:
also, why a boolean value is written as a string?

Reply:
will fix that

Reply:
Done!